docs-source/content/userguide/introduction/introduction.md

```diff
@@ -16,10 +16,9 @@ Detailed instructions are available [here]({{< relref "/userguide/managing-opera
 
 ### Operator prerequisites
 
-For the current production release 2.5.0:
+For the current production release 2.6.0:
 
-* Kubernetes 1.13.5+, 1.14.8+, and 1.15.7+ (check with `kubectl version`). Not supported on Kubernetes 1.16 or later; see note below.
-  See note below for OpenShift.
+* Kubernetes 1.14.8+, 1.15.7+, 1.1.6.0+, 1.17.0+ (check with `kubectl version`).
```

Comment:
Should 1.18.0+ be added?

Reply:
Yes, if we are supporting it. I'll add it.

Reply:
Done.